massa-models/src/operation.rs

```diff
@@ -1273,6 +1273,75 @@ impl Deserializer<Vec<SecureShareOperation>> for OperationsDeserializer {
     }
 }
 
+/// Execution status of an operation
+/// Wrap an optional bool.
+#[derive(Debug, Default)]
+pub struct OperationExecutionStatus {
+    /// None means the operation has not been executed yet
+    /// false the function exited with an error
+    /// true the function succeeded
+    pub status: Option<bool>,
+}
+
+impl OperationExecutionStatus {
+    /// ctor
+    pub fn new(status: Option<bool>) -> Self {
+        Self { status }
+    }
+}
+
+/// Serializer for bool
+#[derive(Debug, Default)]
+pub struct BoolSerializer {}
```

Comment:
This is a more general structure I think it could be saved in an other folder

Reply:
done